assets/svg/help.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="#1d1d1d" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's avoid using hardcoded fills on svgs for simple single color icons. In this case we should default to `currentColor` (I think this has to be on the respective `path` anyways, no?)
```suggestion
<svg width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```